Tools/clinic/clinic.py

```diff
@@ -4400,7 +4399,7 @@ def bad_node(self, node):
                 # mild hack: explicitly support NULL as a default value
                 if isinstance(expr, ast.Name) and expr.id == 'NULL':
                     value = NULL
-                    py_default = 'None'
+                    py_default = '<unspecified>'
```

Comment:
I'd also caution against the use of words like "unspecified" or "undefined" because that can imply to a reader that the behavior is arbitrary and up to the Python implementation when not supplied.  We do define these behaviors, they just require reading further documentation.

`<see docs>` might be better?  the brackets make it clear that it isn't an actual Python value no matter what text goes in there.

Reply:
It does not matter what invalid syntax is used here. The end user will never see it. This is the simplest way to make the insect module not reporting the wrong result. The effect is the same as there is not any signature (as was before using Argument Clinic).

Reply:
What do you think about `<unrepresentable>` or `<nonrepresentable>`?

(This will not be seen by an average Python user, it is only for core developers)